recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 build:
   skip: true  # [py<38 or python_impl == 'pypy']
   script: {{ PYTHON }} -m pip install . -vv
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  script_env:
    - CMAKE_GENERATOR=Ninja
```

Reply:
See https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process, `CMAKE_GENERATOR` is by default Visual Studio project files on Windows.